vector/src/main/java/im/vector/app/features/home/HomeActivity.kt

```diff
@@ -217,7 +217,7 @@ class HomeActivity :
         roomListSharedActionViewModel = viewModelProvider[RoomListSharedActionViewModel::class.java]
         views.drawerLayout.addDrawerListener(drawerListener)
         if (isFirstCreation()) {
-            if (vectorFeatures.isNewAppLayoutEnabled()) {
+            if (vectorPreferences.isNewAppLayoutEnabled()) {
```

Comment:
I think we want to unlock the drawer in the else block, since this is now a runtime setting.

Reply:
Good catch!